packages/postcss-normalize-whitespace/src/index.js

```diff
@@ -9,7 +9,9 @@ function reduceCalcWhitespaces(node) {
   if (node.type === 'space') {
     node.value = ' ';
   } else if (node.type === 'function') {
-    node.before = node.after = '';
+    if (!['var', 'env'].includes(node.value.toLowerCase())) {
```

Comment:
Let's add `constant` (it is old version `env`) + simple test

Reply:
@evilebottnawi  I couldn't find example for `constant` to test upon. Can you share a link for that?
I can see postcss plugins for that or using other processors.

Reply:
just change `env` to `constant` in safari and you can see (just add test where instead `env` you use `constant`)

Reply:
Ok

Reply:
@evilebottnawi done 👍